pkg/api/v1/schema.go

```diff
@@ -32,6 +32,20 @@ import (
  ATTENTION: Rerun code generators when comments on structs or fields are modified.
 */
 
+// Represents a disk created on the cluster level
+// --
```

Comment:
One `-` is missing. Three `---` will make everything after it not appear in the api docs.

Reply:
Fixed